### DISCUSSION TITLE
GitHub page will load on my iPhone 8 but not a new model

### DISCUSSION THREAD

**Discussion by ayemedia21** [community/community#131755] [Mobile]
I have created a QR code with page that I created in VS. When I scan the code on my iPhone 8 it will pull up. When someone else scans it on their phone the page is blank. Here is the link of my code https://github.com/ayemedia21/Final-MG-menu

and here is the link for the actual page what will also load on my Mac. https://ayemedia21.github.io/Final-MG-menu/

any help would be greatly appreciated. I am completely new to this and only learned what little I know from YouTube.

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @ayemedia21!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the Mobile category. I've gone ahead and moved it for you. Good luck!

**Reply by p5hzehxa** [reply]
Tariq Al-Jaarat
